src/server/mod.rs

```diff
@@ -53,6 +53,12 @@ struct LiftService<T, B> {
     _pd: PhantomData<B>,
 }
 
+#[derive(Debug)]
+struct LiftServiceFuture<Fut, B> {
```

Comment:
can we rename this generic to `F` to be consistent with the rest of the code base?

Reply:
fixed in force-pushed commit.